README.rst

```diff
@@ -117,6 +133,73 @@ Sending messages in bulk
 Sending messages in bulk makes use of the bulk mechanics offered by GCM and APNS. It is almost always preferable to send
 bulk notifications instead of single ones.
 
+Multiple mobile applications
+----------------------------
+
+In order to use multiple mobile applications on the same server, you should provide some method to associate the application IDs
+and correspondent application keys or certificates.
+
+If your are planning to use some not big and static number of mobile applications, you can use a static dictionary directly in
+your settings file. Use ``APNS_CERTIFICATES`` and ``GCM_API_KEYS`` settings to store this dictionary for APNS and GCM
+correspondently. The ``APNS_CERTIFICATE`` and ``GCM_API_KEY`` settings are used as defaults.
+
+.. code-block:: python
+
+	PUSH_NOTIFICATIONS_SETTINGS = {
+		"GCM_API_KEY": "<your default application api key>",
+		"GCM_API_KEYS": {
+		    "<application ID 1>":"<your api key 1>",
+		    "<application ID 2>":"<your api key 2>",
+		    ...
+		},
+		"APNS_CERTIFICATE": "/path/to/your/default/certificate.pem",
+		"APNS_CERTIFICATES": {
+		    "<application ID 1>":"/path/to/your/certificate1.pem",
+		    "<application ID 2>":"/path/to/your/certificate2.pem",
+		    ...
+		}
+	}
+
+If your are planning to use dynamic and/or big number of mobile applications, you can use a dynamic access to
+the database table containig records describing applications. Use ``APNS_CERTIFICATES_MODEL``
+and ``GCM_API_KEYS_MODEL`` settings to describe model(s) containing reflection
+of application IDs to the correspondent values. The ``APNS_CERTIFICATE`` and ``GCM_API_KEY`` settings are used
+as defaults.
+
+Let say, the application ``applications`` contains a model ``ApplicationModel`` which contains three fields to
+reflect an application ID to the correspondent application credentials:
+
+.. code-block:: python
+
+    class ApplicationModel(models.Model):
+        application_id = models.CharField(max_length=64,unique=True)
+        gcm_api_key = models.TextField(null=True,blank=True)
+        apns_certificate = models.FileField(null=True,blank=True)
+
+Then settings for the application should look like:
+
+.. code-block:: python
+
+	PUSH_NOTIFICATIONS_SETTINGS = {
+		"GCM_API_KEY": "<your default application api key>",
+		"GCM_API_KEYS_MODEL": {
+		    "model":"applications.ApplicationModel",
```

Comment:
Just read https://github.com/nnseva/django-push-notifications/blob/master/README.rst#multiple-mobile-applications to understand better what is implemented in a patch